test/FakeMuxPort.h

```diff
@@ -31,31 +31,31 @@
 #include "FakeDbInterface.h"
 #include "FakeLinkProber.h"
 
-namespace test
+namespace test 
 {
 
-class FakeMuxPort: public ::mux::MuxPort
-{
+class FakeMuxPort : public ::mux::MuxPort {
 public:
     FakeMuxPort(
         std::shared_ptr<FakeDbInterface> dbInterface,
-        common::MuxConfig &muxConfig,
-        std::string &portName,
+        common::MuxConfig& muxConfig,
+        std::string& portName,
         uint16_t serverId,
-        boost::asio::io_service &ioService
-    );
+        boost::asio::io_service& ioService);
     virtual ~FakeMuxPort() = default;
 
     void activateStateMachine();
 
-    const link_manager::ActiveStandbyStateMachine::CompositeState& getCompositeState() {return getLinkManagerStateMachine()->getCompositeState();};
-    link_prober::LinkProberStateMachine& getLinkProberStateMachine() {return getLinkManagerStateMachine()->getLinkProberStateMachine();};
-    mux_state::MuxStateMachine& getMuxStateMachine() {return getLinkManagerStateMachine()->getMuxStateMachine();};
-    link_state::LinkStateMachine& getLinkStateMachine() {return getLinkManagerStateMachine()->getLinkStateMachine();};
+    std::shared_ptr<link_manager::ActiveStandbyStateMachine> getActiveStandbyStateMachinePtr() { return mActiveStandbyStateMachinePtr; }
+    const link_manager::ActiveStandbyStateMachine::CompositeState& getCompositeState() { return getActiveStandbyStateMachinePtr()->getCompositeState(); };
+    link_prober::LinkProberStateMachine& getLinkProberStateMachine() { return getActiveStandbyStateMachinePtr()->getLinkProberStateMachine(); };
+    mux_state::MuxStateMachine& getMuxStateMachine() { return getActiveStandbyStateMachinePtr()->getMuxStateMachine(); };
+    link_state::LinkStateMachine& getLinkStateMachine() { return getActiveStandbyStateMachinePtr()->getLinkStateMachine(); };
 
-    bool getPendingMuxModeChange() {return getLinkManagerStateMachine()->mPendingMuxModeChange;};
-    common::MuxPortConfig::Mode getTargetMuxMode() {return getLinkManagerStateMachine()->mTargetMuxMode;};
+    bool getPendingMuxModeChange() { return getActiveStandbyStateMachinePtr()->mPendingMuxModeChange; };
+    common::MuxPortConfig::Mode getTargetMuxMode() { return getActiveStandbyStateMachinePtr()->mTargetMuxMode; };
```

Comment:
Do we want to add extra space after left bracket? (This is not big deal though...)

Reply:
We could have this discussed in this PR: https://github.com/Azure/sonic-linkmgrd/pull/28, to have a more specific demand of the coding format.